ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -513,6 +513,19 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private
+- as: ibmcloud-ipi-workers-rhel8-p2-f14
```

Comment:
When we are having the job, it has some duplicated coverage on `ibmcloud-ipi-p2-f7` and `ibmcloud-ipi-private-fips-p2-f7`, especially `ibmcloud-ipi-p2-f7`, I suggest to update `ibmcloud-ipi-p2-f7` and `ibmcloud-ipi-private-fips-p2-f7` to f14 together.

Reply:
Updated. Thanks.